scripts/build_emscripten.sh

```diff
@@ -35,6 +35,7 @@ else
 fi
 
 # solbuildpackpusher/solidity-buildpack-deps:emscripten-13
-docker run -v "$(pwd):/root/project" -w /root/project \
+docker run -v "$(pwd):/emsdk/project" -w /emsdk/project \
+    --user "$(id -u):$(id -g)" \
```

Comment:
```suggestion
    --user "1000:1000" \
```
?

That's the user-id and group-id of the ``emscripten`` user in the docker image, so this way the user actually exists... and it's less prone to non-determinism (not sure there's any way the user/group ID could actually affect anything, but no need to introduce the potential for variance if there's no need).